.golangci.yml

```diff
@@ -87,7 +81,6 @@ linters:
   - gocyclo
   - gofmt
   - goimports
-  - gomnd
```

Comment:
shouldn't we enable `mnd`?

Reply:
`gomnd` [has been desecrated](https://github.com/golangci/golangci-lint/issues/4767#issuecomment-2135208130) - replaced by `mnd` which is declared here now: https://github.com/bacalhau-project/bacalhau/pull/4332/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R35 (line 35 of this PR)

Reply:
You still need to re-declare it here, right? Line 35 is the linter setting, but all linters are disabled except the ones listed here

Reply:
I was initially confused as to why it ran without me setting it - I have modifed the pre-commit script to respect the linter config in the repo: https://github.com/bacalhau-project/bacalhau/pull/4332/commits/7404e375d09be15015438a2f10d6a17d18b83783